multiprofit/psfmodel_utils.py

```diff
@@ -0,0 +1,105 @@
+# This file is part of multiprofit.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import gauss2d.fit as g2f
+
+import numpy as np
+from typing import List, Optional
+
+from multiprofit.limits import limits_ref
+
+
+def make_psf_source(
+    sigma_xs: Optional[List[float]] = None,
+    sigma_ys: Optional[List[float]] = None,
+    rhos: Optional[List[float]] = None,
+    fracs: Optional[List[float]] = None,
+    transforms: dict[str, g2f.TransformD] = None,
+    limits_rho: g2f.LimitsD = None,
+) -> g2f.Source:
+    if limits_rho is None:
+        limits_rho = limits_ref['rho']
+    if sigma_xs is None:
+        sigma_xs = [1.5, 3.0] if sigma_ys is not None else sigma_ys
+    if sigma_ys is None:
+        sigma_ys = sigma_xs
+    n_gaussians = len(sigma_xs)
+    if not n_gaussians > 0:
```

Comment:
In this entire function, please remove the `not`s and replace with different (in)equality operators.

Reply:
I'm not sure I really understand the objection here. It's true that `x == 0` is equivalent when making a comparison to a `len` result in Python, but otherwise it's standard practice to write comparisons like that because it guards against negative or nan values as well.

Reply:
That's a good point, I hadn't thought of that.